### PR TITLE
Fix loongarch support

### PR DESCRIPTION
**Pull request by HecaiYuan**
Add Ioongarch platform support. Welcome to your comments. @luofalei @zywang-sirius

### CODE REVIEW

**Comment by HecaiYuan** [comment]
@zywang-sirius @luofalei

**Review by yinshiyou — APPROVED** [review]

**Review by yinshiyou — COMMENTED** [review]
lgtm.

**Comment by yinshiyou** [comment]
according to the latest CMakelist.txt in src/, uavs3e can only build on aarch64/arm64 and x86 which support sse and avx2.
This patch adapts to loongarch, hope it can be accepted.

**Comment by luofalei** [comment]
LGTM.

File: test/utest.c [hunk 1]
```diff
@@ -57,7 +57,7 @@
 #define FRAME_LOGLEVEL              LOG_LEVEL_1
 
 #ifndef _WIN32
-#if defined(__APPLE__)
+#if defined(__APPLE__) || defined(__loongarch64)
```
Comment (xen0n) [inline test/utest.c]: Consider using just `__loongarch__` here, as the code enabled by the switch doesn't seem to care about whether it's 64-bit or not.

Comment (HecaiYuan) [reply test/utest.c]: OK

**Comment by zzl360** [comment]
good job，hope it will  work on mips64el platform soon

**Review by jinboson — APPROVED** [review]

**Comment by HecaiYuan** [comment]
This patch adapts to loongarch, hope it can be accepted.@luofalei @dujiangpku @zywang-sirius @zzl360

**Review by yinshiyou — COMMENTED** [review]
LGTM

**Review by yinshiyou — APPROVED** [review]

**Pull request merged**